.gitignore

```diff
@@ -11,3 +11,4 @@ Homestead.json
 Homestead.yaml
 npm-debug.log
 yarn-error.log
+.gitignore
```

Comment:
Please do not ignore .gitignore. it will mean all the folders that shouldn't get to the version control gets send there @saidabdul80